src/server/controllers/Grbl/GrblController.js

```diff
@@ -625,6 +645,10 @@ class GrblController {
             if (alarm) {
                 // Grbl v1.1
                 this.emit('serialport:read', `ALARM:${code} (${alarm.message})`);
+                this.emit('electron-error', {
```

Comment:
This is one of the ones to keep - but I'd maybe make the type different to differentiate between GRBL_ERROR and GRBL_ALARM